patches/spectre-console/0002-Remove-Mandelbrot-example-due-to-license.patch

```diff
@@ -0,0 +1,103 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
I think Patrik was going to remove this from the repo tonight, so if we stall a day we may just be able to update the submodule and remove this patch + remove my PackageReference-commenting patch.

Reply:
Oh, is a new release planned soon then?

Reply:
Not a new release, but there have only been [9 commits to `main` since the 0.48.0](https://github.com/spectreconsole/spectre.console/compare/0.48.0...main) release was tagged (at time of writing), and the majority of those have been build-infra related, so I could see an argument for us bumping the submodule to the commits from today that introduced the PackageReference Condition and removed the incompatibly-licensed example file.

Reply:
Our preference is to only pin to a release commit to ensure quality as established by the repo maintainer.